src/config.ts

```diff
@@ -1,61 +1,44 @@
 import fs from "fs";
 import shell from "shelljs";
-
 import log from "./utils/logging";
+import { Config } from "./types";
+import dotenv from "dotenv";
 
-import { Config, Language } from "./types";
-import LANGUAGES from "./utils/languages";
-
-require("dotenv").config();
+dotenv.config();
 
-if (
-  !process.env.CROWDIN_KEY ||
-  !process.env.CROWDIN_PROJECT_NAME ||
-  !process.env.CROWDIN_LANGUAGES
-) {
+if (!process.env.CROWDIN_LANGUAGES) {
```

Comment:
can you add a check for the other variables as well?

Reply:
Ahh, good you spotted this! Removed the old vars, but forgot to add the new ones 😅